app/src/main/java/org/mozilla/fenix/settings/SyncPreferenceView.kt

```diff
@@ -108,11 +109,13 @@ class SyncPreferenceView(
      */
     private fun updateSyncPreferenceNeedsReauth() {
         syncPreference.apply {
-            summary = context.getString(R.string.preferences_passwords_sync_logins_reconnect)
+            isSwitchWidgetVisible = false
 
-            setOnPreferenceClickListener {
+            title = notLoggedInTitle
```

Comment:
From my above comment, I think this makes sense:

s/notLoggedInTitle/loggedOffTitle